docs/Using Fleet/MDM-macOS-setup.md

```diff
@@ -43,9 +45,9 @@ spec:
   mdm:
     end_user_authentication:
       identity_provider_name: "Okta"
-      entity_id: 123
-      issuer_url: "https://example.com"
-      metadata_url: "https://example.com"
+      entity_id: "https://fleetserver.com"
```

Comment:
@noahtalerman I think so long as it is unique, it doesn't matter whether it is an int or string. I changed it to be the fleetserver so that it was more descriptive.